frontend/packages/operator-lifecycle-manager/src/components/modals/installplan-preview-modal.tsx

```diff
@@ -25,11 +25,11 @@ const InstallPlanPreview: React.FC<InstallPlanPreviewModalProps> = ({ cancel, st
         />
       </ModalTitle>
       <ModalBody>
-        <CopyToClipboard value={safeDump(JSON.parse(stepResource.manifest))} />
+        <CopyToClipboard value={safeDump(JSON.parse(stepResource.manifest || ''))} />
```

Comment:
this isn't really safe. JSON.parse could throw an uncaught exception here. We should fix this while here.